scripts/o3de/tests/ExportScripts/test_export_source_built_project.py

```diff
@@ -0,0 +1,1296 @@
+            mock_ctx.args += ['-gpfp', gp]
+        
+        for sp in serverfile_patterns:
+            mock_ctx.args += ['-spfp', sp]
+        
+        for pp in project_patterns:
+            mock_ctx.args += ['-pfp', pp]
+        
+        verify_seedlists = []
+        for sl in seedlists:
+            mock_ctx.args += ['-sl',  sl]
+            verify_seedlists += [pathlib.Path(sl)]
+        
+        verify_seedfiles = []
+        for sf in seedfiles:
+            mock_ctx.args += ['-sf', sf]
+            verify_seedfiles += [pathlib.Path(sf)]
+
+        args = export_standalone_parse_args(mock_ctx, test_export_config)
+
+        assert 'level_names' in args and getattr(args, 'level_names')  == levelnames
+
+        assert 'game_project_file_patterns_to_copy' in args and getattr(args, 'game_project_file_patterns_to_copy') == gamefile_patterns
+
+        assert 'server_project_file_patterns_to_copy' in args and getattr(args, 'server_project_file_patterns_to_copy') == serverfile_patterns
+
+        assert 'project_file_patterns_to_copy' in args and getattr(args, 'project_file_patterns_to_copy') == project_patterns
+    
+        assert 'seedlist_paths' in args and getattr(args, 'seedlist_paths') == verify_seedlists
+        assert 'seedfile_paths' in args and getattr(args, 'seedfile_paths') == verify_seedfiles
+
+        export_standalone_run_command(mock_ctx, args, test_export_config, mock_logger)
+
+        mock_export_func.assert_called_once_with(ctx=mock_ctx,
+                        selected_platform='mac',
+                        output_path=tmpdir/'output',
+                        should_build_tools=False,
+                        build_config='profile',
+                        tool_config='profile',
+                        seedlist_paths=verify_seedlists,
+                        seedfile_paths=verify_seedfiles,
+                        level_names=levelnames,
+                        game_project_file_patterns_to_copy=gamefile_patterns,
+                        server_project_file_patterns_to_copy=serverfile_patterns,
+                        project_file_patterns_to_copy=project_patterns,
+                        asset_bundling_path=tmpdir / 'assets/bundling/paks',
+                        max_bundle_size=mock_bundle_size,
+                        should_build_all_assets=False,
+                        fail_on_asset_errors=False,
+                        should_build_game_launcher=True,
+                        should_build_server_launcher=True,
+                        should_build_unified_launcher=True,
+                        engine_centric=False,
+                        allow_registry_overrides=False,
+                        tools_build_path=tmpdir / 'tool_builds',
+                        launcher_build_path=pathlib.PurePath('LauncherBuilds/Packages'),
+                        archive_output_format='none',
+                        monolithic_build=False,
+                        logger=mock_logger)
```

Comment:
nitpick: Can you add a newline to the end of the file
```suggestion
                        logger=mock_logger)

```